requirements.txt

```diff
@@ -10,7 +10,7 @@ PyGithub==1.58.2
 pyperclip==1.8.2
 PySide6==6.4.3
 pytz==2023.3
-requests==2.28.2
+requests==2.31.0
```

Comment:
Have you checked all in-app functionality functionality related to this module upgrade on all platforms?

Reply:
Found some more info about the timezones' thing, refer to discord chat, about time formatting, see link for more information pytz==2023.3 https://pypi.org/project/pytz/ 
Requests==2.28.2 to 2.31.0 version update, see link for more information, I was reading between the versions, this is optional, referring to this part of the text: Users who rely on defining their proxy credentials in the URL are strongly encouraged to upgrade to Requests 2.31.0+ to prevent unintentional leakage and rotate their proxy credentials once the change has been fully deployed.

Users who do not use a proxy or do not supply their proxy credentials through the user information portion of their proxy URL are not subject to this vulnerability https://requests.readthedocs.io/en/latest/community/updates/#release-history